### PR TITLE
💅 Add date picker

### PR DESCRIPTION
**Pull request by eliascodes**
Related to #39 

### Changes
* Install `react-datepicker`
* Overwrite `react-datepicker` styles with custom ones to match initial design
* Separate definition for `Label` component
* Allow custom select box to accept child elements
* Group font related enums into one object

No unit tests here because the main dependency of `react-datepicker` doesn't work nicely with `jsdom` (see https://github.com/FezVrasta/popper.js/issues/478)

### CODE REVIEW

File: dashboard-app/src/components/DatePicker/styles.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.react-datepicker__header {
+  background-color: white;
```
Comment (astroash) [inline dashboard-app/src/components/DatePicker/styles.css]: Can this be a `ColourEnum`?

File: dashboard-app/src/components/DatePicker/styles.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  width: 90%;
+  margin: 0 auto;
+}
+
+.react-datepicker__month {
+  margin: 0.2em;
+}
+
+.react-datepicker__month--selected {
+  background-color: #707070;
```
Comment (astroash) [inline dashboard-app/src/components/DatePicker/styles.css]: `ColourEnum`

File: dashboard-app/src/components/DatePicker/styles.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+.react-datepicker__month-text.react-datepicker__month--selected:hover {
+  background-color: rgba(112, 112, 112, 0.8);
```
Comment (astroash) [inline dashboard-app/src/components/DatePicker/styles.css]: `ColourEnum`

File: dashboard-app/src/components/Select.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  }
+`;
+
+export const Option = styled.option`
+  font: ${Fonts.family.default};
+  font-size: ${Fonts.size.normal};
+`;
+
+const ErrorText = styled.span`
+  color: red;
```
Comment (astroash) [inline dashboard-app/src/components/Select.tsx]: `ColourEnum`

**Comment by astroash** [comment]
Can you also check console errors? Not sure whether mine are caused my faux props:

Screenshot 2019-05-01 at 10 45 50

Screenshot 2019-05-01 at 10 45 34

File: dashboard-app/src/components/DatePicker/styles.css (hunk 1 above)
Comment (eliascodes) [reply dashboard-app/src/components/DatePicker/styles.css]: Its a CSS file.

File: dashboard-app/src/components/DatePicker/styles.css (hunk 2 above)
Comment (eliascodes) [reply dashboard-app/src/components/DatePicker/styles.css]: Its a CSS file

**Comment by eliascodes** [comment]
> Can you also check console errors? Not sure whether mine are caused my faux props

I don't get those errors, I just get warnings about iterated elements not having keys (which I've just fixed in the most recent commit). I also get a warning about the `isActive` prop which is something from the navbar, so I wont address that here.

**Pull request merged**